src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -110,5 +112,22 @@ public Task BytesMetrics()
 
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/periodic-export-stats", "GET")]
```

Comment:
Backup? I think it was renamed from Periodic Backup to Periodic Export in 3.0 to avoid confusion?